### ISSUE TITLE
Storage add-on v2: Update the product price data

### ISSUE THREAD

**Issue by southp** [Automattic/wp-calypso#97223]
### Details

This depends on #97220. 

Update the prices listed in `marketing/product-prices-data.php` ~to make sure they are reflected in the price listing tool.~

Correction: Relooking into the code, that price data is actually used in LOHP and the landing pages. Thus, the Landpack part of #97221 will depend on this.

**Comment by aneeshd16** [comment]
@southp, I believe we can close this?

**Comment by southp** [comment]
@aneeshd16 Hmm, the extended storage add-on prices seem to work without updating all the `prices` array here: https://github.a8c.com/Automattic/wpcom/blob/trunk/wp-content/lib/marketing/product-prices-data.php#L128. Do you know if it is really used in anywhere? If not, we can close this issue.

**Comment by aneeshd16** [comment]
Oh I see what you are referring to. That data is used here: 36fc0-pb. I'll update this in a separate PR.
